modules/generate_dyn_universe.py

```diff
@@ -119,7 +119,7 @@ def AddBasesToSystem (faction,sys):
         tn =[]
         for i in range(len(shiplist)):
             tn+=[ (shiplist[i],nums[i])]
-        fg_util.AddShipsToFG(fg_util.BaseFGInSystemName (sys),faction,tn,sys)
+#        fg_util.AddShipsToFG(fg_util.BaseFGInSystemName (sys),faction,tn,sys)
```

Comment:
So this code no longer generates any bases at all? It's left to some other code elsewhere to do it?

Reply:
It might be improperly loading the systems and not properly detecting that they've already been loaded

Reply:
There seems to be two parallel functions for creating bases. 
This one can't name the bases properly, and sometimes ends up making up to seven bases in a system. 
The bases that are generated with the other function end up in .vegastrike/sectors, work properly, and seem to be the right amount of bases, for me at least.

Reply:
I think it would be a good idea to have some other people play-test as well, before merging.

Reply:
@stephengtuggy I fully agree with you, this does need a lot of play testing. You simply can't just take my word for it that the gameplay is better or worse. What you enjoy in a video game could be completely different that what I would like. 
Let's consider this PR. All it does is comment out a command that is causing issues. To test it you could very easily do the same edit on your test machine and go for a spin in the VS universe.
You could just accept the PR and have the same effect, and if you hate it you could roll it back. Or, make a single character edit and be back to the old behavior. 
I have a serious issue with the old behavior, in that the names of the installations is corrupted. This is still something I want to fix. When I do crack the nut, I would like to make this a configure option for users to turn on or off as they like. 
In the meantime, I'm zooming around in a VS universe that has proper names everywhere, and the relative sparseness of land-able bases makes the user jump around more, and get to explore more. I've been playing in this type of universe for the last few hours, and I quite like the experience. 

I would like you all to test this to see if you agree, please.

Reply:
Regarding play testing: there's a way to checkout PRs locally to build them (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally); alternatively one could pull down anyone else's fork either directly or via an additional remote.

There's some cases like this where it will be necessary to do those kinds of things. Hopefully as we get better understanding of the code and can mature stuff for maintenance we can make those the exception and not the rule.